Src/IronPython/Runtime/Operations/PythonOps.cs

```diff
@@ -470,6 +470,9 @@ public static partial class PythonOps {
         }
 
         public static object Is(object x, object y) {
+            if(x is bool && y is bool) {
```

Comment:
What about reusing https://github.com/IronLanguages/main/pull/1332 instead of having something different? Unless this fixes something else...